assets/sass/patterns/atoms/dismiss-button.scss

```diff
@@ -0,0 +1,44 @@
+@import "../../definitions";
+
+.dismiss-button {
+
+  .call-to-action & {
```

Comment:
`dismiss-button` is not a pattern so should not have its own sass file. The separate implementations in this file, `.call-to-action` and `.info-bar`, are very different and there is no need to group them together.